env-development/main.tf

```diff
@@ -1 +1,80 @@
-# development.tf
+# DEVELOPMENT
+
+data "aws_availability_zones" "available" {}
+
+data "aws_region" "current" {
+  current = true
+}
+
+data "aws_region" "us-east-1" {
+  name = "us-east-1"
+}
+
+data "aws_region" "us-west-1" {
+  name = "us-west-1"
+}
+
+data "aws_route53_zone" "zone" {
+  name = "${var.fqdn}." # feel free to make this lookup more sophisticated
+}
+
+data "aws_vpc" "vpc" {
```

Comment:
Can I put in a vote for using ID instead of name? ala  - https://github.com/FitnessKeeper/terraform-consul-cluster/blob/master/main.tf#L2   One benefit of using ID, over name is that if/when we  change the VPC names this will  mean a smaller impact on our spikes...  or at least the spikes will keep working correctly.

Reply:
hm, i do like the idea of being able to supply a human-friendly name...

Reply:
For terraform spikes, I'm somewhat split on human-friendly names, for module inputs though, I really think we should stick with VPC ID's.  Do you feel like the user friendly names are worth the extra effort in refactoring if they change?

Reply:
in an ideal world i would like to never look up a VPC ID ever, since they are supremely unmemorable to me; i take your point about refactoring and resource churn though.  we can switch to VPC ID, but i wince a little.

Reply:
Yeah, I feel ya... I guess the other option is to just change any spike that does a lookup using VPC name...  I mean in theory they shouldn't change often.

Reply:
i have changed this variable over to `vpc_id` :)